docs/Developers/Review.rst

```diff
@@ -75,13 +77,23 @@ The review process for coala is as follows:
 
     .. note::
 
-        Only one acknowledgment is needed per commit i.e ``ack commit_SHA``.
+          Only one acknowledgment is needed per commit i.e ``ack commit_SHA``.
+
+
+   * In case a member is reviewing it:
+
+    - Looks good to me better known as ``LGTM`` in case the commit is ready
```

Comment:
It would be better if you put `Looks good to me` between inverted coma.

Reply:
Done!